openmls/src/framing/codec.rs

```diff
@@ -210,3 +210,66 @@ impl tls_codec::Size for MlsMessage {
         }
     }
 }
+
+impl Size for MlsCiphertextContent {
+    fn tls_serialized_len(&self) -> usize {
+        self.content.tls_serialized_len() +
+           self.signature.tls_serialized_len() +
+            self.confirmation_tag.tls_serialized_len() +
+            // Note: The padding is appended as a "raw" all-zero byte slice
+            // with length `length_of_padding`. Thus, we only need to add
+            // this length here.
+            self.length_of_padding as usize
```

Comment:
Let's not use unsafe casts. You can do `usize::from` here.

Reply:
Widening casts from `u16` to `usize` are safe. That's why I used `as` here and `try_from` in another place. But I like the idea to avoid `as` in general and can use `from`.

Reply:
With this reasoning we can put everything in an `unsafe` block because we know it's safe. `as` is generally a bad idea for casts that are safe as you have a `from` function in these cases.